src/jellyfin.ts

```diff
@@ -0,0 +1,58 @@
+import express from 'express';
+import { ParamsDictionary } from 'express-serve-static-core';
+
+import type { Request, Response } from 'express';
+import { unmonitorMovie } from './radarr.js';
+import { unmonitorEpisode } from './sonarr.js';
+
+const { JELLYFIN_PORT = '9898' } = process.env;
+
+export function startJellyfinUnmonitor() {
+  console.log(
+    `Unmonitoring for jellyfin on /jellyfin with port: ${JELLYFIN_PORT}`,
+  );
+
+  const app = express();
+
+  app.post(
+    '/jellyfin',
+    express.json(),
+    (
+      req: Request<ParamsDictionary, unknown, JellyfinApiResponse>,
+      res: Response,
+    ) => {
+      const { Item, Series } = req.body;
+
+      if (Item.Type === 'Episode' && Item.UserData.Played) {
```

Comment:
Can you just have an early return for `!Item.UserData.Played` and convert this to a switch?